packages/react-core/src/components/Card/examples/CardHeaderWraps.tsx

```diff
@@ -0,0 +1,26 @@
+import { Card, CardBody, CardHeader, Button, CardTitle, Flex, FlexItem } from '@patternfly/react-core';
+
+export const CardBasic: React.FunctionComponent = () => (
+  <Card ouiaId="BasicCard">
+    <CardHeader hasWrap>
+      {' '}
+      <Flex columnGap={{ default: 'columnGapXl' }} justifyContent={{ default: 'justifyContentSpaceBetween' }}>
+        <FlexItem>
+          <CardTitle>This is a longer card title that takes up more space</CardTitle>
+        </FlexItem>
+        <FlexItem>
+          <Button variant="primary" ouiaId="Primary">
+            Primary action
+          </Button>
+          <Button variant="secondary" ouiaId="Secondary">
+            Secondary action
+          </Button>
+          <Button variant="tertiary" ouiaId="Tertiary">
+            Tertiary action
+          </Button>
+        </FlexItem>
+      </Flex>
+    </CardHeader>
```

Comment:
So for the content inside CardHeader, we can keep the CardTitle inside it as a child. The Buttons we want to pass to CardHeader's `actions` prop (and remove all of the Flex components):

```suggestion
    <CardHeader
      hasWrap
      actions={{
        hasNoOffset: true,
        actions: [
          <Button variant="primary" ouiaId="Primary">
            Primary action
          </Button>,
          <Button variant="secondary" ouiaId="Secondary">
            Secondary action
          </Button>,
          <Button variant="tertiary" ouiaId="Tertiary">
            Tertiary action
          </Button>
        ]
      }}
    >
      <CardTitle>This is a longer card title that takes up more space</CardTitle>
    </CardHeader>
```

Reply:
Didn't realise `actions` are this useful. Will make the necessary changes